src/mumble/SettingsKeys.h

```diff
@@ -169,7 +169,7 @@ const SettingsKey CHANNEL_EXPANSION_MODE_KEY           = { "channel_expansion_mo
 const SettingsKey CHANNEL_DRAG_MODE_KEY                = { "channel_drag_mode" };
 const SettingsKey USER_DRAG_MODE_KEY                   = { "user_drag_mode" };
 const SettingsKey ALWAYS_ON_TOP_KEY                    = { "always_on_top" };
-const SettingsKey ASK_ON_QUIT_KEY                      = { "ask_on_quit" };
```

Comment:
We should keep the `ASK_ON_QUIT_KEY` for the migration code that we have to add. If a user has a value for `ask_on_quit` in their JSON, Mumble should automatically convert that into the new `connected_quit_behavior` setting.